### ISSUE TITLE
AMSmath subequation enviroment

### ISSUE THREAD

**Issue by jmlopez-rod** [mathjax/MathJax#760]
I hate to be a bother since I know that there are many issues and a lot of work to be done, but there is one thing that I would love to have for mathjax. Will there be support in the future for the AMSmath subequation enviroment? Has it been considered yet or it is too hard to implement at the moment? For now, my latex documents contain blocks like this:

```
\begin{subequations}
 \label{eq1}
 \begin{align}
  a & = b \label{eq11} \\
  c & = d \label{eq12}
 \end{align}
\end{subequations}
```

I can get rid of the subequation enviroment and just have a new number for each of the subequations but now in my text I have to get rid of every reference to the main label `eq1`. Is there any workaround for now to the lack of support to subequations?

**Comment by tdehaeze** [comment]
Any news on that?

**Comment by dpvc** [comment]
@tdehaeze, I'm not sure what "that" refers to.  The subequations environment?  The issue with multiple labels for for align environment?  I believe that the latter is fixed (see #691).  There is a v2 implementation for a (limited) subequations environment listed in [this post](https://groups.google.com/d/msg/mathjax-users/8YxzxF-m_hM/CJCkceuoCgAJ).

**Comment by shengchao-lin** [comment]
I'd like to +1 on adding support for subequations environment.
Or any syntax that allows automatic grouping tags (e.g., (3.1), (3.2)) for equations.

**Comment by dpvc** [comment]
@shengchao-lin, [this issue](https://github.com/mathjax/MathJax/issues/2427) discusses implementing section numbers using the v3 `tagformat` package.  A similar approach could be used to get sub equation numbers.  Something like the code in the post linked in myth previous message could be implemented in v3 as well, though the API is different.  but the same idea could be used.
